lib/scoped_search/query_builder.rb

```diff
@@ -83,16 +83,20 @@ def build_find_params(options)
       return find_attributes
     end
 
-    def order_by(order, &block)
+    def find_field_for_order_by(order, &block)
       order ||= definition.default_order
       return nil if order.blank?
```

Comment:
This used to short-circuit exit the order_by method but no longer does as it's in a different method now.  Maybe add a check in `order_by` after calling `find_field_for_order_by` to `return nil if field.nil?`